build/style.css

```diff
@@ -100,3 +85,109 @@ strong {
 	opacity: 0.6;
 	border-bottom-color: #bb5536;
 }
+
+h2.trouble {
+	float: right;
+	padding: 20px 0px;
+	color: #a9a9a9;
+	text-transform: uppercase;
+}
+
+
+/****** END HEADER *****/
+
+/**** MAIN CONTENT ****/
+
+
+.section .wrapper {
+	padding: 42px 20px;
+}
+
+.article {
+	display: flex;
+}
+
+.main {
+    flex: 3 0 0;
+    margin-right: 50px;
+}
+
+div.main h1 {
+	color: #b30000;
+}
+
+
+/************SIDE BAR ********/
+
+
+.side-bar {
+    flex: 1 0 0;
+		padding-top: 30px;
+}
+
+.side-bar .mod {
+	border: 1px solid #ddd;
+}
+
+.side-bar h2 {
+	color: #333;
+	padding: 16px;
+	font-weight: 600;
+	line-height: 1em;
+	background: #fafafa;
+	font-size: 18px;
+	margin: 0;
+	border-bottom: 1px solid #ddd;
+}
+
+.side-bar ul {
+	margin: 0;
+	padding: 0;
+}
+
+.side-bar li {
+	margin: 0;
+	padding: 0;
+	border-bottom: 1px solid #ddd;
+	list-style: none;
+}
+
+.side-bar li:last-child {
+	border-bottom: 0;
+}
+
+.side-bar a {
+	padding: 12px 16px;
+	display: block;
+	text-decoration: none;
+	color: #333;
+	font-weight: 400;
+}
+
+.side-bar a:hover {
+	text-decoration: underline;
+	background: #fafafa;
+	color: #cc0000;
+}
+
+
+/******** END SIDE BAR****/
+
+/****** Footer ******/
+
+
+.footer {
+	background: #4d4d4f;
+	color: #a9a9a9;
+	padding: 20px;
+
+}
+
+
+.icon{
```

Comment:
This is one where you will want to be a little more specific. You don't want all your icons across the entire site to be floated right. Change this to something like `.footer .social .icon { ... }`